appveyor.yml

```diff
@@ -0,0 +1,33 @@
+version: "{build}"
+
+# Source Config
+
+clone_folder: c:\gopath\src\github.com\docker\cli
+
+# Build host
+
+environment:
+  GOPATH: c:\gopath
+  DEPTESTBYPASS501: 1
+  GOVERSION: 1.9
+
+init:
+  - git config --global core.autocrlf input
+
+# Build
+
+install:
+  # Install the specific Go version.
+  - rmdir c:\go /s /q
+  - appveyor DownloadFile https://storage.googleapis.com/golang/go%GOVERSION%.windows-amd64.msi
+  - msiexec /i go%GOVERSION%.windows-amd64.msi /q
+  - set Path=c:\go\bin;c:\gopath\bin;C:\Program Files (x86)\Bazaar\;C:\Program Files\Mercurial\%Path%
+  - go version
+  - go env
+
+build: false
+deploy: false
+
+test_script:
+  - go build github.com/docker/cli/cmd/docker
+  - for /f "" %%G in ('go list github.com/docker/cli/...') do ( go test %%G & IF ERRORLEVEL == 1 EXIT 1)
```

Comment:
I think we can still use bash, ex: https://github.com/stedolan/jq/blob/master/appveyor.yml, https://github.com/containerd/containerd/blob/master/.appveyor.yml

Reply:
another example: https://github.com/gotestyourself/gotestyourself/pull/69/files#diff-180360612c6b8c4ed830919bbb4dd459R22